packages/teams-js/src/public/meeting.ts

```diff
@@ -518,4 +518,25 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.meetingReactionReceived', handler);
   }
+  /**
+   * nested namespace for Meeting Service
+   *
+   * @beta
+   */
+  export namespace meetingAppShareButton {
```

Comment:
The prefix "meeting" here is unnecessary since callers will already have to write this code:

`meeting.meetingAppShareButton`. Would be better as `meeting.appShareButton`.

Do you anticipate any other appShareButton functions being added in the future?